### ISSUE TITLE
Use AppGroupID that is different from "group.\(this.hostAppBundleIdentifier)"

### ISSUE THREAD

**Issue by vlytvyne** [KasemJaffer/receive_sharing_intent#107]
Hi. Code in `ios/Share Extension/ShareViewController.swift` contains something like following
```
let hostAppBundleIdentifier = "com.kasem.sharing"
...
let userDefaults = UserDefaults(suiteName: "group.\(this.hostAppBundleIdentifier)")
```

Which assumes that AppGroupID will definitely be looking like `group.bundle.id`, what I have in my app is `group.bundle.id.share`. I suppose that is the reason why using this plugin when I press on share and pick my app it does open but doesn't get any data in. I tried to replace `"group.\(this.hostAppBundleIdentifier)"` with `this.fullAppGroupID` in `ios/Share Extension/ShareViewController.swift`, but still don't get the data in the app, I guess it might be hardcoded in few more places but not sure where. Can you please provide more info on that?

**Comment by maciej-szulc** [comment]
Hey, have you managed to make it work? I'am struggling with it right now ;/

**Comment by vlytvyne** [comment]
> Hey, have you managed to make it work? I'am struggling with it right now ;/

Yes, I found a fix. What you will have to do is to copy the package to your project and refer to it in pubspec as following
```
receive_sharing_intent:
    path: './receive_sharing_intent'
```

Now go to `your_project/receive_sharing_intent/ios/Classes/SwiftReceiveSharingIntentPlugin.swift` and replace your group id here `let groupId = "your_group_id"` in function `handleUrl`.

Then go to `your_project/ios/Share Extention/ShareViewController.swift` and declarate your group id `let groupId = "your_group_id"` then replace all `"group.\(this.hostAppBundleIdentifier)"` with `groupId`.

Now everything should work!
